tests/suites/test_suite_psa_crypto_util.function

```diff
@@ -0,0 +1,44 @@
+/* BEGIN_HEADER */
+#include <stdint.h>
+#include <test/helpers.h>
+#include <mbedtls/psa_util.h>
+#include <mbedtls/asn1write.h>
```

Comment:
Why is that one needed? Is that for the error codes? Then perhaps it should be included by `psa_util.h` because otherwise it means `psa_util.h` is not giving you everything you need to use the functions it declares.